sites/friday/src/get-started/express/_index.md

```diff
@@ -1,4 +1,435 @@
+[//]: # (<--->)
+
+[//]: # (+++)
+
+[//]: # (title=Gitlab repository)
+
+[//]: # (+++)
+
+[//]: # ([//]: # &#40;TODO this will not be part of the MVP, cf. https://platformsh.slack.com/archives/C05DSRYLK2P/p1691062519908969&#41;)
+[//]: # (*Pros*: For the rest of this tutorial, you will use the normal Git Flow to push your source code to your Git repository and Github Integration hook will automatically deploy your corresponding Upsun environment with latest code updates.)
+
+[//]: # (*Cons*: ??)
+
+[//]: # (<--->)
+
+[//]: # (+++)
+
+[//]: # (title=Bitbucket repository)
+
+[//]: # (+++)
+
+[//]: # ([//]: # &#40;TODO this will not be part of the MVP, cf. https://platformsh.slack.com/archives/C05DSRYLK2P/p1691062519908969&#41;)
+[//]: # (*Pros*: For the rest of this tutorial, you will use the normal Git Flow to push your source code to your Git repository and Github Integration hook will automatically deploy your corresponding Upsun environment with latest code updates.)
+
+[//]: # (*Cons*: ??)
+
+{{< /codetabs >}}
+
+## Create a new project
+The next step is to create a project on {{% vendor/name %}}.
+To do this, you can either use the {{% vendor/name %}} CLI or the {{% vendor/name %}} console—details on how to do this can be found below.
+
+{{< note title="Remember" >}}
+At the end of either using the {{% vendor/name %}} CLI or the Console section, please copy your new **projectId**, we’ll use it in the next section: link your local project to your {{% vendor/name %}} project.
+{{< /note >}}
+
+
+{{< codetabs >}}
++++
+title=using the CLI
++++
+To create a new project with the Upsun CLI, use the following command:
+```shell
+$ upsun project:create --plan flexible
+Selected organization: ACME Inc. on Upsun (acme-inc-on-upsun) (by default)
+Creating a project under the organization ACME Inc. on Upsun (acme-inc-on-upsun)
+
+* Project title (--title)
+Default: Untitled Project
+> My First Upsun Project
+
+* Region (--region)
+The region where the project will be hosted
+  [org.recreation.plat.farm] org.recreation.plat.farm
+> org.recreation.plat.farm
+
+Default branch (--default-branch)
+The default Git branch name for the project (the production environment)
+Default: main
+>
+
+The estimated monthly cost of this project is: 26,45 €
+
+Are you sure you want to continue? [Y/n] y
+Your Upsun project has been requested (subscription ID: 12345678)
+
+The Upsun Bot is activating your project
+
+      ▄     ▄
+      ▄█▄▄▄█▄
+    ▄██▄███▄██▄
+    █ █▀▀▀▀▀█ █
+       ▀▀ ▀▀
+
+The project is now ready!
+xy52zopcxxd7u
+
+  Region: org.recreation.plat.farm
+  Project ID: xy52zopcxxd5rf
+  Project title: My First Upsun Project
+  URL: https://console.upsun.com/01h14dh7t32qd3nn1mcrd3x82e/xy52zopcxxd5rf
+  Git URL: xy52zopcxxd5rf@git.org.recreation.plat.farm:xy52zopcxxd7u.git
+```
+
+{{< note >}}
+When creating a new project using the {{% vendor/name %}} CLI, your local source code will automatically be linked to your newly created {{% vendor/name %}} project.
```

Comment:
I change the order again, 
So now, at the beginning of the tutorial, you need to do the `git init` and so, during `project:create` it will ask the question to set the remote to the local Git repo. 
i just add a note at the beginning of the Set Remote section to let the reader know that he would do this section only if he answered no to the question during project creation. 
Is it ok?